src/index.tsx

```diff
@@ -40,6 +40,7 @@ type QRProps = {
   marginSize?: number;
   imageSettings?: ImageSettings;
   title?: string;
+  alt?: string;
```

Comment:
I commented on the issue, so we can continue there. But practically, all this is doing is changing the type that gets exported. Is that what you're looking for. The code is already going to pass `alt` along so it will be in the DOM.